treelib/tree.py

```diff
@@ -118,7 +118,7 @@ def _clone(
         identifier: Optional[str] = None,
         with_tree: bool = False,
         deep: bool = False,
-    ):
+    ) -> "Tree":
```

Comment:
I would normally use `-> Tree` here and import `__future__.annotations` but I'm not sure if that's supported by all versions of Python that treelib supports, so this felt a little bit safer.